Missing semicolons and rephrase (#568)
openqasm/openqasm: source/language/openpulse.rst

    // driveframe1.time == driveframe2.time == 13ns
  }

> Moreover, ``defcal`` blocks have an implicit ``barrier`` on every frame entered in the block e.g.

Reviewer: ```suggestion
Moreover, ``defcal`` blocks have an implicit ``barrier`` on every frame used by the block. For example:
```
